iocore/eventsystem/UnixEventProcessor.cc

```diff
@@ -491,3 +489,36 @@ EventProcessor::spawn_thread(Continuation *cont, const char *thr_name, size_t st
 
   return e;
 }
+
+bool
+EventProcessor::has_tg_started(int etype)
+{
+  return thread_group[etype]._started == thread_group[etype]._count;
+}
+
+bool
+EventProcessor::has_tgs_started(std::vector<int> etypes)
```

Comment:
Where is this used? If the values passed are `constexpr`, talk to be about `std::initializer_list` and the Leif asploding stuff you can do with it in a run time efficient manner.

Reply:
I was just testing things and this seemed kind of useful at the time and forgot about it later. lol